drafts/draft-snell-multicodec-00.txt

```diff
@@ -0,0 +1,336 @@
+   carefully, as they describe your rights and restrictions with respect
+   to this document.
+
+
+
+
+Snell, et al.            Expires August 27, 2020                [Page 1]
+
+Internet-Draft                Multiformats                 February 2020
+
+
+Table of Contents
+
+   1.  Introduction  . . . . . . . . . . . . . . . . . . . . . . . .   2
+   2.  Codec Registrations . . . . . . . . . . . . . . . . . . . . .   2
+   3.  Security Considerations . . . . . . . . . . . . . . . . . . .   5
+   4.  IANA Considerations . . . . . . . . . . . . . . . . . . . . .   5
+   5.  Normative References  . . . . . . . . . . . . . . . . . . . .   5
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .   5
+
+1.  Introduction
+
+   TBD
+
+   In this document, the key words "MUST", "MUST NOT", "REQUIRED",
+   "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY",
+   and "OPTIONAL" are to be interpreted as described in [RFC2119].
+
+2.  Codec Registrations
+
+   +--------------------+-------------+----------+---------------------+
+   | Name               | Tag         | Code     | Description         |
+   +--------------------+-------------+----------+---------------------+
+   | ip4                | multiaddr   | 0x04     |                     |
+   | tcp                | multiaddr   | 0x06     |                     |
+   | dccp               | multiaddr   | 0x21     |                     |
+   | ip6                | multiaddr   | 0x29     |                     |
+   | ip6zone            | multiaddr   | 0x2a     |                     |
+   | path               | namespace   | 0x2f     | Namespace for       |
+   |                    |             |          | string paths.       |
+   |                    |             |          | Corresponds to / in |
+   |                    |             |          | ASCII.              |
+   | multiaddr          | multiformat | 0x32     |                     |
+   | dns                | multiaddr   | 0x35     |                     |
+   | dns4               | multiaddr   | 0x36     |                     |
+   | dns6               | multiaddr   | 0x37     |                     |
+   | dnsaddr            | multiaddr   | 0x38     |                     |
+   | protobuf           | serializati | 0x50     | Protocol Buffers    |
+   |                    | on          |          |                     |
+   | cbor               | serializati | 0x51     | CBOR                |
+   |                    | on          |          |                     |
+   | raw                | ipld        | 0x55     | raw binary          |
+   | rlp                | serializati | 0x60     | recursive length    |
+   |                    | on          |          | prefix              |
+   | bencode            | serializati | 0x63     | bencode             |
+   |                    | on          |          |                     |
+   | dag-pb             | ipld        | 0x70     | MerkleDAG protobuf  |
```

Comment:
I would rename all occurrences from `MerkleDAG` to just `DAG`. I think `MerkleDAG` is a historical name and I would prefer if we get rid of all references.

Reply:
Agree on removing MerkleDAG as a ref. Alternatively, rename to "IPLD DAG protobuf"